profile/README.md

```diff
@@ -15,16 +15,36 @@
 
 # Local Setup Instructions 👋
 
-1. Create directory named "Traverse" in local machine.
-2. Cd into the directory.
+## Clone + Setup Repos
+1. Create a directory named "Traverse" somewhere in your local machine.
+2. cd into the directory.
 3. git clone each repo.
 4. For nodejs repos, use node version 16+/
 5. Run npm install on nodejs projects.
-6. Download neo4j desktop client: https://neo4j.com/deployment-center/#desktop
+
+## Database Installation
+
+### neo4j
+1. Download neo4j desktop client: https://neo4j.com/deployment-center/#desktop
     - May have to scroll down to desktop section.
-    - Install version 1.5.8 
-7. Create env file for main-service.
```

Comment:
Add localhost:8080 storage service url as env. 
Also put JWT_WEB_TOKEN. Dev token can be exposed, choose a new token for dev.

Reply:
How do we get the web token again?

Reply:
Token added for traverse-ui and main-service